lavalink/Client.py

```diff
@@ -129,23 +133,37 @@ def unregister_hook(self, func):
         async with self.http.get(node.rest_uri + quote(query), headers={'Authorization': node.password}) as res:
             return await res.json(content_type=None)
 
-    async def get_player(self, guild_id: int):
+    async def get_player(self, guild_id: int, create: bool = True):
         try:
             await asyncio.wait_for(self.nodes.ready.wait(), timeout=10.0)
         except asyncio.TimeoutError:
             raise NoNodesAvailable
         for node in self.nodes:
             if guild_id in node.players:
-                log.debug("Found player in cache.")
+                log.debug('Found player in cache.')
                 return node.players[guild_id]
-        log.debug("Player not in cache")
+        log.debug('Player not in cache')
+        if not create:
+            return None
         guild = self.bot.get_guild(guild_id)
         if guild is None:
-            log.debug("Couldn't find the guild.")
+            log.debug('Couldn\'t find the guild.')
             return self.nodes.nodes[0].players.get(guild_id)
-        log.debug("Getting new player from geo.")
+        log.debug('Getting new player from geo.')
         return self.nodes.get_by_region(guild)
 
+    async def garbage_collection(self):

```

Comment:
I think garbage collection should be opt-in behavior

Reply:
I'll meet you in the middle and remove the GC for now aka until I find a viable solution.

Reply:
👌